tst/com/amazon/ion/benchmark/DataGeneratorTest.java

```diff
@@ -294,6 +306,40 @@ public void testParseBenchmark() throws Exception {
         }
     }
 
+    /**
+     * Test whether the detecting regression process can return the expected result.
+     * In this unit test we use an Ion file which contain regression on [·gc.alloc.rate] as input to test the detectRegression method.
+     * @throws Exception if error occur when reading Ion data.
+     */
+    @Test
+    public void testRegressionDetecting() throws Exception {
```

Comment:
(Apologies if this comment appears twice; it looks like it was lost when I tried to post it)

This verifies that a regression is detected when it is present, but we should also test that a regression is _not_ detected when it is _not_ present. You'll be able to reuse most of the code from this test.

Reply:
> (Apologies if this comment appears twice; it looks like it was lost when I tried to post it)
> 
> This verifies that a regression is detected when it is present, but we should also test that a regression is _not_ detected when it is _not_ present. You'll be able to reuse most of the code from this test.

No problem. Additional unit test will be added.